modules/installing-ocp-agent-inputs.adoc

```diff
@@ -159,4 +164,25 @@ You must provide the rendezvous IP address when you do not specify at least one
 <2> Optional: Host configuration. The number of hosts defined must not exceed the total number of hosts defined in the `install-config.yaml` file, which is the sum of the values of the `compute.replicas` and `controlPlane.replicas` parameters.
 <3> Optional: Overrides the hostname obtained from either the Dynamic Host Configuration Protocol (DHCP) or a reverse DNS lookup. Each host must have a unique hostname supplied by one of these methods.
 <4> Enables provisioning of the Red Hat Enterprise Linux CoreOS (RHCOS) image to a particular device. It examines the devices in the order it discovers them, and compares the discovered values with the hint values. It uses the first discovered device that matches the hint value.
-<5> Optional: Configures the network interface of a host in NMState format.
\ No newline at end of file
+<5> Optional: Configures the network interface of a host in NMState format.
+
+ifdef::pxe-boot[]
+
+. Optional: To create an iPXE script, add the `bootArtifactsBaseURL` to the `agent-config.yaml` file:
++
+[source,yaml]
+----
+apiVersion: v1alpha1
+kind: AgentConfig
+metadata:
+  name: sno-cluster
+rendezvousIP: 192.168.111.80
+bootArtifactsBaseURL: <asset_server_URL>
+----
++
+Where `<asset_server_URL>` is the URL of the server your PXE assets will be uploaded to.
```

Comment:
Maybe we should clear it that users need to upload the generated PXE files i.e. initrd, kernel, rootfs and  optional iPXE script to this location. The current wording makes it look agent-installer will automatically upload the files to asset_server_url.

Reply:
This makes it clear later here https://github.com/openshift/openshift-docs/pull/65762/files#diff-9930080c8c807d0352e77a1b118dfbe67742aee0ac7b4cd286620be38358a9b1R37 but may be can also be made clearer here https://github.com/openshift/openshift-docs/pull/65762/files#diff-f6e661459f8da7184d7d71a7c1e3f7031cfeb0749d90c1f98d57c2869c460bd3R183

Reply:
Thanks for this suggestion, I should have made this more user focused in the first place.